generators/writing/templates/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+	"watch": ["src/**/*.ts"],
+	"execMap": {
+		"ts": "tslint -p tsconfig.json -c tslint.json && ts-node --files --seed --typeCheck"
```

Comment:
wouldn't this reseed the db on every code change?

Reply:
It wouldn't without NODE_ENV being set beforehand, either in the environment or in the command itself.

The generated scripts produce dual scripts such as `script` and `script:seed`.